core-ui/src/main/java/com/google/samples/apps/nowinandroid/core/ui/NewsResourceCard.kt

```diff
@@ -267,9 +267,35 @@ fun NewsResourceShortDescription(
 
 @Composable
 fun NewsResourceTopics(
-    newsResource: NewsResource
+    topics: List<Topic>,
+    modifier: Modifier = Modifier
 ) {
-    TODO()
+    LazyRow(
```

Comment:
Just confirming the choice of a scrollable `LazyRow` here, instead of a `FlowRow` or something similar?

Reply:
yup, design in figma has it as a scrollable row right now.

Reply:
I think this should not be a `LazyRow`, as these are not recommended when the number of elements is small (too much overhead). Instead, you could simply use a `Row` with `horizontalScroll` modifier. @bentrengrove to confirm.

Reply:
It would depend on how many total there are, how many are displayed on screen at once and how heavy each item is to compose. You would have to benchmark it for a proper answer but in general it does switch over to Lazy being better pretty quick but if the majority of items are always displayed on the screen then Row is probably better.

Reply:
most items have just 1-3 elements in the row, so updated to use `Row`